### ISSUE TITLE
tmux settings have changed

### ISSUE THREAD

**Issue by protist** [jcfaria/Vim-R-plugin#191]
vim-r's help suggests the following in `~/.tmux`:
```
set -g mode-mouse on
set -g mouse-select-pane on
set -g mouse-resize-pane on
```
However, this fails with an error in the [latest tmux 2.1-1](https://stackoverflow.com/questions/11832199/tmux-set-g-mouse-mode-on-doesnt-work). These settings have been removed in tmux, and replaced with the following.
```
set -g mouse on
```
vim-r's help should be updated to reflect this.

**Comment by jalvesaq** [comment]
Thanks for reporting the issue!

**Comment by protist** [comment]
No worries. I also just noticed that if tmux isn't already started, the temporary configuration file at `/tmp/r-plugin-$USER/tmux.conf` also contains these issues, which prevents tmux/r from starting. Is there a workaround to prevent vim-r from sourcing this temporary file in the meantime? Thanks in advance.

**Comment by jalvesaq** [comment]
I believe it is fixed now.

**Comment by jranke** [comment]
I can confirm that the issue (which I had when not starting tmux before starting vim) is fixed in the master branch. Thanks a lot!

**Comment by protist** [comment]
Thanks. I'm using the stable release, so I was looking for a workaround in this. However, if it's not possible, I'll just wait. Cheers.

**Comment by jalvesaq** [comment]
@jranke Thanks for the feedback! I am still using Tmux 2.0.

@protist I released a new version.

**Comment by protist** [comment]
@jalvesaq Thanks, vim-r 1.2.7 with vimcom 1.2-7 seems to be mostly working. That is, it works with tmux/R launched in Konsole (the default terminal for me) and xterm (if the latter is specified in `let vimrplugin_term = "xterm"`). However, I can't get it to work in `terminator` any more. I just get the message `Waiting vimcom loading...`, then `The package vimcom wasn't loaded yet.` In 1.2.6, even though tmux was broken for me (as per the original report), terminator would still open. Now it doesn't even launch at all.

**Comment by jalvesaq** [comment]
I need to reproduce the bug to be able to fix it, but I have just installed terminator 0.97 and it works.

**Comment by protist** [comment]
I spent a while trying to troubleshoot this with different permutations.
* I created a fresh Arch install in Virtualbox, but cannot reproduce this bug here.
* This install used a minimal `~/.vimrc`, so I copied that to my normal system, but hit the bug anyway.
* I thought perhaps one of the many plugins might conflict, so I disabled them with `vim -u ~/.vimrc test.Rmd`, but again I hit the bug.

Is there anything else that might be different? `~/.Rprofile` and `~/.tmux.conf` are identical (although I believe both are run downstream of terminator launching).

**Comment by jalvesaq** [comment]
If the external terminal emulator (Terminator) is not starting at all, it could be due to an error in the command that calls it. What I usually do is to add a line to the function `StartR_ExternalTerm` to store the command in a global variable.  Currently, it would be on line 759 of *r-plugin/common_global.vim*:
```vim
let g:the_opencmd = opencmd
```
Then, after trying to start R, do `:echo the_opencmd` and try to run the command manually in a terminal emulator.

**Comment by protist** [comment]
I spent a little while on this, but can't work it out. (It's academic for me anyway, since I'd prefer running vim-r from within tmux.) FWIW here is what I tried, but I've given up trying to reproduce it.

`opencmd` looked similar in both installs, although it only executed fine in the clean Virtualbox environment. On my normal system, I tried executing the command itself (i.e. everything past the `-x` flag in `terminator`), which opened up the tmux session perfectly. Hence, it only seems to choke when opening a new terminator session.

Similarly, if I quit terminator, and run vim-r in konsole, it will launch terminator fine. It seems to be the presence of an existing terminator instance that causes it to choke. In the clean Virtualbox instance, it opens up a new instance fine.

I thought perhaps my terminator config was to blame somehow, so I copied it to my clean Virtualbox install. However, even with this, the first test worked fine.

Anyway, as I say, this bug doesn't directly affect me, so I've given up trying to isolate it. If you have a specific test, then I'm happy to try it, but otherwise this can stay closed. Thanks for your help.
